### ISSUE TITLE
Input Field Missing Testability Hook

### ISSUE THREAD

**Issue by jturfanbasis** [webforj/webforj#1085]
### Describe the bug

The input element with class .element--input currently lacks a reliable attribute (like aria-label, placeholder, or data-testid) for automation testing. This makes it difficult to target the input in Playwright and ensure stable, non-flaky tests.

### Reproduction

1. Navigate to http://localhost:8080/elementinputtext.
2. Inspect the input field rendered in the DOM.
3. Notice the following markup:
`<input dwc-id="11" class="element--input" style="">`

No aria-label, name, placeholder, or data-testid attributes are present.

<img width="1916" height="1031" alt="Image" src="https://github.com/user-attachments/assets/f6d280d8-56fe-4863-b3c8-6de897e2cac1" />

### Deployment

WAR (Default)

### System Info

25.03 -snapshot
